students/myszko_pawel/lesson_02_flow_control/06_Chess board - same color.py

```diff
@@ -0,0 +1,21 @@
+# Given two cells of a chessboard. If they are painted in one color, print the word YES, and if in a different color - NO.
+# The program receives the input of four numbers from 1 to 8, each specifying the column and row number,
+# first two - for the first cell, and then the last two - for the second cell.
+
+x1 = int(input())
+y1 = int(input())
+x2 = int(input())
+y2 = int(input())
+
+matrix_black = []
+matrix_white = []
+for i in range(8):
+    for j in range(8):
+        if ((j%2 == 1) and (i%2 == 1)) or ((j%2 == 0) and (i%2 == 0)):
```

Comment:
Wewnętrzne nawiasy są zbędne, kolejność operatorów gwarantuje, że `and` zostanie wykonany po `==`.